### DISCUSSION TITLE
2023-07-20

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#113] [daily_meeting]
### 어제의 회고

- 6시에 모여서 체크아웃하기

### 오늘의 할 일

- 기능 추가 및 목요일까지 완성된 기능들을 배포

- 발표자들은 나머지와 내용 협의해서 발표 준비

### 언젠가 해야할 일
기획 변경된 이유 정리
DNS 어떻게 할지
UX 저자 워크숍 사전 미션(~7/25 화) -> 준이 제공한 내용 참고해 데모 끝나고 월요일에 하기로 결정

**Comment by woosung1223** [comment]
### 체크인
10점 - 점수 깎일 이유가 없어서 10점!

### 체크아웃
9점 - PPT 만드는게 은근 힘이 들어서 -1점!

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 어제까지 최소 기능 구현이 끝나서 기분은 좋은데 정리할 게 산더미라 1점, 오늘 오는데 날씨가 너무 습해서 1점 깎았습니다.

### 체크아웃
9점 - 저희가 2차 스프린트 목표까지 잡았던 바는 다 한거 같아 1점 올랐습니당.

**Comment by aak2075** [comment]
### 체크인

10점 - 조금 피곤하지만 일정도 잘 진행되고 있어요 화이팅합시다🚀

### 체크아웃

10점 - 삽질 많이 했지만 그래도 다 해결해서 다행입니다👍

**Comment by nlom0218** [comment]
### 체크인

10점 - 페이지 개발을 할 수 있어서 행복해요

### 체크아웃

9점 - 데이터 다루기 힘드러요

**Comment by yeopto** [comment]
### 체크인

4점 - 3일동안 네시간씩 자서 너무 피곤하네요..

### 체크아웃

7점 - 한번 고생해서 만들어 놨더니 페이지 금방 만들 수 있는거 같아서 기분이 좋다.

**Comment by woo-jk** [comment]
### 체크인
3점 - 자는 사이 휴대폰이 꺼져서 지각했다

### 체크아웃
5점 - 내일 발표가 떨린다
